java/core/src/test/java/sleeper/core/testutils/printers/TablesPrinter.java

```diff
@@ -24,24 +24,39 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+/**
+ * A utility class to print matching and differing strings generated from a table name.
+ * Intended to be used for approval tests.
+ */
 public class TablesPrinter {
 
     private TablesPrinter() {
     }
 
-    public static String printForAllTables(Collection<String> tableNames, Function<String, String> tablePrinter) {
+    /**
+     * Prints matching and differing strings generated using a table name. When multiple tables generate the
+     * same string, the output is only printed once, along with a summary of how many tables generated the same
+     * output string. This method is useful for when you want to assert whether the output for all tables are identical.
+     * <p>
+     * Note that this will not output table names.
+     *
+     * @param  tableNames   the collection of table names
+     * @param  tablePrinter the function that generates a printed line from a table name
+     * @return              a string containing output strings generated by tables
+     */
+    public static String printForAllTablesExcludingNames(Collection<String> tableNames, Function<String, String> tablePrinter) {
         Map<String, List<String>> tableNamesByPrintedValue = tableNames.stream()
                 .collect(Collectors.groupingBy(tablePrinter));
         List<Map.Entry<String, List<String>>> printedSortedByFrequency = tableNamesByPrintedValue.entrySet().stream()
                 .sorted(Comparator.comparing(entry -> entry.getValue().size()))
                 .collect(Collectors.toUnmodifiableList());
-        ToStringPrintStream printer = new ToStringPrintStream();
+        ToStringPrintWriter printer = new ToStringPrintWriter();
         PrintStream out = printer.getPrintStream();
 
         for (Map.Entry<String, List<String>> entry : printedSortedByFrequency) {
-            String printed = entry.getKey();
-            List<String> printedForTables = entry.getValue();
-            int frequency = printedForTables.size();
+            String printedLine = entry.getKey();
+            List<String> tablesThatPrintThisLine = entry.getValue();
+            int frequency = tablesThatPrintThisLine.size();
```

Comment:
These variable names are misleading. It's the printed string, and the tables that print that string. It's not one line.